core/iwasm/common/wasm_runtime_common.h

```diff
@@ -592,9 +592,6 @@ wasm_runtime_call_indirect(WASMExecEnv *exec_env, uint32 element_indices,
 bool
 wasm_runtime_create_exec_env_singleton(WASMModuleInstanceCommon *module_inst);
 
-WASMExecEnv *
```

Comment:
Had better not remove the declaration here, just add WASM_RUNTIME_API_EXTERN like others:
```C
WASM_RUNTIME_API_EXTERN WASMExecEnv *
wasm_runtime_get_exec_env_singleton(WASMModuleInstanceCommon *module_inst);
```

Reply:
i moved this to wasm_export.h. do you mean to have it in both headers?

Reply:
i think i got it now. although i feel it's a weird convention, i updated the patch to follow it. thank you.

Reply:
Yes, prefer to have it in both files.